internal/modules/staking/tx.py

```diff
@@ -3,7 +3,7 @@
 import re
 from utils import exec_command, execute_tx_by_type
 from modules.staking.query import (
-    get_public_key,
+    get_validator_pubkey,
```

Comment:
This is still confusing. How about the following?
```suggestion
    fetch_validator_pubkey_from_node,
```

This would give better clarity on what parameters to pass.